vagrant/provisioning/roles/privacy/templates/arkcase-portal-server.yaml

```diff
@@ -0,0 +1,40 @@
+logging.level.web: DEBUG
+
+# server.port=8080
+# server.servlet.context-path=/${portal.applicationName}
+
+# Property used for enable/disable Spring security. For authenticated portal comment this property.  
+#spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration,org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration
+
+
+portal.externalConfiguration: true
+portal.host={{ external_host }}
+portal.port=
+portal.uiApplicationUrl=https://{{ external_host }}/privacy
+portal.uiApplicationHost=https://{{ external_host }}
```

Comment:
This file is also a funny mix of yaml and properties syntax.

Reply:
Yep. They are wrong. I'll fix them.